tcf_website/templates/reviews/review.html

```diff
@@ -50,6 +36,22 @@ <h5><strong>{{ review.course }}</strong> — {{ review.instructor.full_name }}</
                 </div>
 
                 <div class="review-footer row mt-3">
+                    <div class="col-sm-8 d-flex align-items-end text-center text-muted flex-wrap mb-2">
+                        <div class="mr-3 mr-sm-4">
+                            <a id="upvote{{review.id}}" href="#loginModal" data-toggle="modal"
+                               class="upvote {% if review.user_vote == 1 %}active{% endif %}">
+                                <i class="fa fa-thumbs-up fa-fw"></i>
+                            </a>
+                            <span id="vote-count" class="voteCount">{{ review.count_votes.upvotes }}</span>
+                        </div>
+                        <div class="mr-3 mr-sm-4">
+                            <a id="downvote{{review.id}}" href="#loginModal" data-toggle="modal"
+                               class="downvote {% if review.user_vote == -1 %}active{% endif %}">
+                                <i class="fa fa-thumbs-down fa-fw"></i>
```

Comment:
I think it's a bit weird that the upvote is red as typically that is negative? Although I guess it is matching reddit style...
![image](https://user-images.githubusercontent.com/26658168/110248067-473f5e80-7f3d-11eb-94ad-583360898762.png)
It also looks a bit too large on my screen.

Reply:
I agree, I will discuss with someone from the design team.